docs/content/getting-started/code-generation.md

```diff
@@ -138,33 +136,33 @@ $ hof gen data.cue schema.cue -T template.txt -T =debug.yaml -O out/
 
 ### Watching for Changes
 
-Use `-w`/`--watch` to watch for changes and re-render output.
+Use `-w`/`--watch` to observe for changes and re-render output.
 
 {{<codeInner title="> terminal" lang="sh">}}
 $ hof gen data.cue schema.cue -T template.txt -T debug.yaml -O out/ --watch
 {{</codeInner>}}
 
-There are extra watch flags if automatic detection doesn't fully work.
+There are extra watch flags if automatic detection doesn't entirely work.
 
 
 ## On Using CUE
 
 <br>
 
-`hof`'s inputs are `cue`'s inputs, or "CUE entrypoints".
+`hof`'s inputs are `cue`'s inputs, or "CUE entry points".
```

Comment:
this space is going to be needed in a lot of places... :/

Reply:
I can bulk edit from entrypoints to entry points 🔨